### DISCUSSION TITLE
Can I just acquire domain for now?

### DISCUSSION THREAD

**Discussion by Preet-Sojitra** [is-a-dev/register#5817] [Q&A] [answered]
Currently I have `.me` domain taken from namecheap and my website is hosted on vercel on this url [https://www.preetsojitra.me](https://www.preetsojitra.me) . This domain will be expiring next month. So for now I want to grab this domain. So can I and also want to know what to put in CNAME record and can I use this domain in vercel?

**Comment by wdhdev** [comment, marked as answer]
You can redirect your domain to your Vercel website.
